sources/australia/au/AGRIblack-and-white25m.geojson

```diff
@@ -4,12 +4,16 @@
         "id": "AGRI-black_and_white-2.5m",
         "url": "https://{switch:a,b,c}.agri.openstreetmap.org/layer/au_ga_agri/{zoom}/{x}/{y}.png",
         "attribution": {
-            "text": "AGRI",
+            "text": "AGRI, Geoscience Australia",
+            "url": "https://data.gov.au/dataset/agri-the-australian-geographic-reference-image"
```

Comment:
there's a comma missing at the end of this line

Reply:
thanks. fixed now